frontend/src/metabase/query_builder/components/view/sidebars/SummarizeSidebar/BreakoutColumnList/BreakoutColumnListItem/BreakoutColumnListItem.module.css

```diff
@@ -60,6 +60,7 @@
     }
 
     .BucketTriggerButton {
+      visibility: visible;
```

Comment:
I accidentally removed this from the original PR https://github.com/metabase/metabase/pull/53961